scripts/cli

```diff
@@ -495,6 +495,11 @@ do
             start
             exit;
         ;;
+        start-be)
+            declare -a INGORED_SERVICES=("premium-api" "premium-job-generator" "premium-python-worker" "community-help-center" "python-worker" "frontend" "discord-ws")
```

Comment:
I think Uros removed these services `premium-api`, `premium-python-worker` and `premium-job-generator`, so no need to add those here